app/Http/Controllers/DocsController.php

```diff
@@ -91,7 +91,10 @@ public function show($version, $page = null)
         $metaKeywords = $pageCustomData['keywords'] ?? self::DEFAULT_META_KEYWORDS;
         $communityNote = $pageCustomData['communityNote'] ?? true;
         $contributors = $pageCustomData['contributors'] ?? false;
-        $contributorsData = explode(',', $contributors);
+        $contributorsData = [];
+        if ($contributors) {
+            $contributorsData = explode(',', $contributors);
+        }
```

Comment:
```suggestion
        $contributorsData = $contributors ? explode(',', $contributors) : [];
```

what do you think? :-)

Reply:
I changed it.